src/fal/planner/tasks.py

```diff
@@ -24,6 +25,14 @@ def execute(self, args: argparse.Namespace, fal_dbt: FalDbt) -> int:
         raise NotImplementedError
 
 
+class GroupStatus(Enum):
+    PENDING = auto()
```

Comment:
did not know about auto

Reply:
Good for when you don't have an exact value.